enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/document/DocumentVerificationService.java

```diff
@@ -131,9 +200,13 @@ private void accept(
             final IdentityVerificationEntity identityVerification,
             final List<DocumentVerificationEntity> documentVerifications,
             final OwnerId ownerId) {
-        documentVerifications.forEach(docVerification ->
-            auditService.audit(docVerification, "Document accepted at final verification for user: {}", identityVerification.getUserId()));
-        identityVerificationService.moveToPhaseAndStatus(identityVerification, DOCUMENT_VERIFICATION_FINAL, ACCEPTED, ownerId);
+
+        final IdentityVerificationPhase phase = identityVerification.getPhase();
+        documentVerifications.forEach(docVerification -> {
+            docVerification.setStatus(DocumentStatus.ACCEPTED);
+            auditService.audit(docVerification, "Document accepted at phase {} for user: {}", phase, identityVerification.getUserId());
+        });
+        identityVerificationService.moveToPhaseAndStatus(identityVerification, phase, ACCEPTED, ownerId);
```

Comment:
[] Movement to another phase and status should be done only for final verification. For first investigation, we have another idea at #515